src/components/organisms/AppRouter/AppRouter.tsx

```diff
@@ -72,6 +72,7 @@ export const AppRouter: React.FC = () => {
     <Router basename="/">
       <Suspense fallback={<LoadingPage />}>
         <Switch>
+          <Route path="/loading" component={LoadingPage} />
```

Comment:
I suppose this was added for testing/developing purposes, please remove it.

Reply:
fixed in https://github.com/sparkletown/sparkle/pull/2005/commits/0f5f808e5b41a355da9aa72c5fc40f4fa7320562